packages/account/src/Components/forms/form-fields/form-input-field.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { FieldInputProps, FormikHelpers, FormikState, Field } from 'formik';
+import { Input } from '@deriv/components';
+
+type FormInputFieldProps = {
+    name: string;
+    optional?: boolean;
+    warn?: string;
+} & React.ComponentProps<typeof Input>;
+
+type TFormInputFieldHelpers = {
+    field: FieldInputProps<string>;
+    form: FormikHelpers<Record<string, string>> & FormikState<Record<string, string>>;
+};
```

Comment:
```suggestion
type TFormInputFieldHelpers = {
    field: FieldInputProps<string>;
    form: FormikHelpers<T> & FormikState<U>;
};
```
If you are providing a wrapper around Formik, then these two need to be generic types that are depending on the field value passed to this component. Doing this will pass the TypeScript check but the component will lose type safety

Reply:
Will make this change